api/c/indigo/src/indigo.cpp

```diff
@@ -51,6 +51,39 @@ CEXPORT const char* indigoVersion()
     return INDIGO_VERSION "-" INDIGO_PLATFORM;
 }
 
+CEXPORT const char* indigoVersionInfo()
+{
+    INDIGO_BEGIN
+    {
+        std::string version = indigoVersion();
+
+        const std::string digits = "0123456789";
+        const auto parse = [&](const auto& shift) { return version.substr(shift, version.find_first_of("-\\")); };
+        const auto slice = [&](const auto& value) {
+            version.erase(0, value.size() + 1);
+            return value;
+        };
+
+        const std::string major_version = slice(version.substr(0, parse(1).find_last_of(digits)));
```

Comment:
This will cause error if minor version consists of more than one digit

Reply:
Fixed.